ui/app/account-detail.js

```diff
@@ -26,11 +26,10 @@ function mapStateToProps (state) {
     accounts: state.metamask.accounts,
     address: state.metamask.selectedAccount,
     accountDetail: state.appState.accountDetail,
-    transactions: state.metamask.transactions,
     network: state.metamask.network,
-    unconfTxs: valuesFor(state.metamask.unconfTxs),
     unconfMsgs: valuesFor(state.metamask.unconfMsgs),
     shapeShiftTxList: state.metamask.shapeShiftTxList,
+    transactions: state.metamask.transactions,
```

Comment:
after reverting the change to the getState function it should be
`transactions: state.metamask.selectedAccountTxList,`